packages/mobile/src/geth/selectors.ts

```diff
@@ -0,0 +1,7 @@
+import { InitializationState } from 'src/geth/reducer'
+import { RootState } from 'src/redux/reducers'
+
+export const isGethConnectedSelector = (state: RootState) =>
```

Comment:
Dupe of the existing selector in geth/reduer.ts

Reply:
Removed from reducer to be consistent with keeping selectors in /selectors